infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/StreamsSnapshotWriter.java

```diff
@@ -241,14 +241,17 @@ public void apply(LogReplicationEntryMsg message) {
             return;
         }
         UUID regularStreamId = opaqueEntry.getEntries().keySet().stream().findFirst().get();
+        UUID shadowStreamId = getShadowStreamUUID(regularStreamId);
+        log.trace("Shadow stream=[{}] for regular stream=[{}]", shadowStreamId, regularStreamId);
 
-        // When an upgrade has been performed and Active's streams to replicate set has streams not
-        // present in Standby's set, they will be dropped for version compatibility consideration
-        if (!regularToShadowStreamId.containsKey(regularStreamId)) {
-            log.warn("Stream {} sent from Active is not expected in Standby. LR could be undergoing a rolling upgrade", regularStreamId);
+        // Filter out noisy streams which already have is_federated flag set to false.
```

Comment:
shouldnt updates to the registry table be processed first?  Otherwise how can we determine which streams are noisy?